docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,286 @@
+<!--docs:
+title: "Material sheets: bottom"
+layout: detail
+section: components
+excerpt: "Bottom sheets are surfaces containing supplementary content that are anchored to the bottom of the screen."
+iconId:
+path: /catalog/material-sheets-bottom/
+-->
+
+# Sheets: bottom
+
+[Bottom sheets](https://material.io/components/sheets-bottom) are surfaces containing supplementary content that are anchored to the bottom of the screen.
+
+![Example bottom sheet: modal bottom sheet](assets/bottom_sheets/SheetsBottom_hero.png)
```

Comment:
replace alt-text with:

Bottom sheet with 4 items, with scrim covering the remaining screen.